xml/System.Numerics/Complex.xml

```diff
@@ -6816,7 +6816,7 @@ Languages that support custom operators can use the <xref:System.Numerics.Comple
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- The default string representation of a complex number displays the number using its Cartesian coordinates in the form `(`*a*`,` *b*`)`, where *a* is the real part of the complex number, and *b* is its imaginary part. Both *a* and *b* are formatted using the general format specifier ("G") and the conventions of the current system culture.
+ The default string representation of a complex number displays the number using its Cartesian coordinates in the form `<`*a*`;` *b*`>`, where *a* is the real part of the complex number, and *b* is its imaginary part. Both *a* and *b* are formatted using the general format specifier ("G") and the conventions of the current system culture.
```

Comment:
@jeffhandley It seems like we should keep the old _and_ new syntax and annotate them with the versions they're applicable to.

Reply:
OK, I can send a follow up PR, @gewarren. Would the following wording be good? Please feel free to offer refinement to it.

```suggestion
 The default string representation of a complex number displays the number using its Cartesian coordinates in the form `<`*a*`;` *b*`>`, where *a* is the real part of the complex number, and *b* is its imaginary part. Both *a* and *b* are formatted using the general format specifier ("G") and the conventions of the current system culture. In .NET 7 and earlier, the form `(`*a*`, `*b*`)` was used.
```

Reply:
```suggestion
 The default string representation of a complex number displays the number using its Cartesian coordinates in the form `<`*a*`;` *b*`>` (or `(`*a*`, `*b*`)` in .NET 7 and earlier versions), where *a* is the real part of the complex number, and *b* is its imaginary part. Both *a* and *b* are formatted using the general format specifier ("G") and the conventions of the current system culture.
```

This would be my recommended wording.